e2e/linter/src/linter.test.ts

```diff
@@ -315,7 +335,32 @@ function updateGeneratedRuleImplementation(
 
         return ts.visitEachChild(node, visit, context);
       }
-      return ts.visitNode(rootNode, visit);
+      /**
+       * Add lib import as a first line of the rule file
```

Comment:
Please could we add the "why" behind this for the benefit of other engineers who weren't directly involved in this PR? It might not be clear why we felt this was important